quantum/quantum.c

```diff
@@ -201,7 +201,7 @@ bool process_record_quantum(keyrecord_t *record) {
   keypos_t key = record->event.key;
   uint16_t keycode;
 
-  #if !defined(NO_ACTION_LAYER) && !defined(STRICT_LAYER_RELEASE)
+  #if !defined(NO_ACTION_LAYER) && defined(PREVENT_STUCK_MODIFIERS)
```

Comment:
This needs to be reverted.  There is no more PSM, as it is now the default behavior. 

```suggestion
  #if !defined(NO_ACTION_LAYER) && !defined(STRICT_LAYER_RELEASE)
```